Source/JavaScriptCore/parser/Parser.cpp

```diff
@@ -2426,7 +2426,7 @@ template <class TreeBuilder> typename TreeBuilder::FormalParameterList Parser<Le
         auto binding = context.createBindingLocation(location, name, position, position, AssignmentContext::DeclarationStatement);
         context.appendParameter(parameters, binding, 0);
         ++parameterCount;
-    };
```

Comment:
I think we should keep this semicolon because it is an assignment.
See https://webkit.org/code-style-guidelines/#other-punctuation, which has "auto a = [] { };"